cmd/gh/main.go

```diff
@@ -336,7 +337,11 @@ func shouldCheckForUpdate() bool {
 	if os.Getenv("CODESPACES") != "" {
 		return false
 	}
-	return updaterEnabled != "" && !isCI() && utils.IsTerminal(os.Stdout) && utils.IsTerminal(os.Stderr)
+	return updaterEnabled != "" && !isCI() && isTerminal(os.Stdout) && isTerminal(os.Stderr)
+}
+
+func isTerminal(f *os.File) bool {
```

Comment:
This function also exists unexported in the `iostreams` package, I had inlined it here from the `utils` package. Should we export it from `iostreams` to prevent this duplication?

Reply:
It's fine like this. I will take care of this when I start dogfooding iostreams stuff from go-gh; thanks!